### ISSUE TITLE
Import tab:  "Refresh" button doesn't work

### ISSUE THREAD

**Issue by regexgit** [jim-easterbrook/Photini#299]
I've imported from a directory, then added photos to this directory, but the refresh button doesn't show these new files. 
Workaround: change directory and go back to 1st directory.

```
Photini 2024.10.0.post15+g76b8666a
  Python 3.9.2 (default, Feb 28 2021, 17:03:44) 
[GCC 10.2.1 20210110]
  python-exiv2 0.16.1, exiv2 0.27.7
  PyQt 6.7.1, Qt 6.7.1, chrome 118.0.5993.220
  system locale fr, locales: fr-FR fr-Latn-FR fr
  styles: Windows, Fusion
  using style: fusion
```

**Comment by jim-easterbrook** [comment]
That "refresh" is to refresh the list of cameras (detected by libgphoto2) - it hadn't occurred to me that it could refresh a directory listing.

**Comment by jim-easterbrook** [comment]
"Refresh" now reloads the file list. Released as version 2024.11.0.
